dataset_models/sdo/aia.py

```diff
@@ -460,12 +465,12 @@ def _get_aia_image(self, filename, directory, previous=0):
         assert previous >= 0, "previous should be a non-negative integer, it is currently " + previous
         assert previous < 5, "previous should not be a very large integer, it is currently " + previous
         if previous == 0:
-            data = feather.read_dataframe(directory + filename)
-            return data.values
+            data = feather.read_dataframe(directory + filename).values.reshape(1024,1024,8)
```

Comment:
Why is this reshape now necessary? Same below.

Reply:
It was sometime ago, but I believe it was necessary for it to work.  I'm not sure why.  For some reason in python3 the shape was all wrong.

Same below.

Reply:
I think we need to visualize this to make sure the new way is good. I am worried about introducing some weird non-coherent expression of the data.